iOS-Email-Client/Controllers/CustomTabsController.swift

```diff
@@ -11,10 +11,33 @@ import Material
 
 class CustomTabsController: TabsController {
     
+    var myAccount: Account!
+    var devicesData = DeviceSettingsData()
+    var generalData = GeneralSettingsData()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         self.navigationItem.title = "SETTINGS"
         self.navigationItem.leftBarButtonItem = UIBarButtonItem(image: #imageLiteral(resourceName: "close-rounded").tint(with: .white), style: .plain, target: self, action: #selector(dismissViewController))
+        self.loadData()
+    }
+    
+    func loadData(){
+        APIManager.getDevices(token: myAccount.jwt) { (error, devices) in
+            guard let myDevices = devices else {
+                return
+            }
+            for device in myDevices {
+                let newDevice = Device.fromDictionary(data: device)
+                guard !self.devicesData.devices.contains(where: {$0.id == newDevice.id && $0.active}) else {
+                    continue
+                }
+                self.devicesData.devices.append(newDevice)
+            }
+            self.generalData.recoveryEmail = "pedro.aim93@gmail.com"
```

Comment:
wat

Reply:
retrieving recoveryEmail is not implemented in the server yet, the same for the recovery email status

Reply:
I'm not sure how getting your devices relate to your recovery email

Reply:
getDevices will soon change into something like getSettings, you could wait for this change in the server but lot of prs are gonna stacks